__mocks__/react-moveable.js

```diff
@@ -19,7 +19,7 @@
  */
 import { forwardRef } from 'react';
 
-// eslint-disable-next-line no-unused-vars
+// eslint-disable-next-line no-unused-vars  -- @todo
```

Comment:
We could probably just disable this rule for all files in `__mocks__`:

https://github.com/GoogleForCreators/web-stories-wp/blob/700032264a04185f9ca2b5232e3fe61a8ca34313/.eslintrc#L315-L322